### PR TITLE
nmon: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/nmon.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Start nmon:
+
+`nmon`
+
+- Save records to file (default args "-s300 -c288" for 300 seconds between refreshing the screen and 288 refreshes):
+
+`nmon -f`
+
+- Save records to file with 30 seconds refreshing gap and 240 refreshes:
```
Comment (agnivade) [inline pages/linux/nmon.md]: What does "240 refreshes" mean ? Does it mean that within a single interval, it will take 240 measurements ?

Comment (jsonbruce) [reply pages/linux/nmon.md]: No. It means the system will take 240 measurements and the gap between each measurement is 30 seconds. So the whole time monitored is 2 hours.
I've drew a illustration to explain this command:

selection_200

Comment (agnivade) [reply pages/linux/nmon.md]: I see. In that case, I would slightly change the description to something like - 
`Save records to file with a total of 240 measurements by taking 30 seconds between each measurement.`

Comment (jsonbruce) [reply pages/linux/nmon.md]: Great, I love this sentence!

**Comment by jsonbruce** [comment]
LGTM. Thanks! @sbrl

**Pull request merged**